### ISSUE TITLE
Policy Regression: breaks the non-scheduled green energy option

### ISSUE THREAD

**Issue by ngardiner** [ngardiner/TWCManager#124]
Thanks to @dschuesae's PR we have picked up a regression with the new policy structure: It removes a feature from the past where setting nonScheduledAmpsMax to -1 would result in green energy tracking instead of non-scheduled amps.

I'd like to fix this a number of ways:

1. Remove the old option from the old webui to avoid confusion. Even when we do fix this, I wouldn't want to do it with a negative amps max value - the reason we've avoided this in the new policy code is because it could easily slip through the cracks and cause some weird behaviour if it was indeed taken as the charge amps value.

2. Add an option to the new webui to allow this. It would be a new boolean value in the settings.json file and could then be set as a policy condition.

3. Add a catch-all track green energy policy rule after non-scheduled charging so that if this boolean sets the behaviour to skip non-scheduled charging, it will track green energy instead.

**Comment by ngardiner** [comment]
Stage 1 of this I have completed - (1) and (2). The old Web UI no longer allows the broken Track green energy option to be set for nonScheduledAmpsMax, but still allows all of the other options. Should be equivalent functionality to today but without the broken option.

I have added a new parameter nonScheduledAction which is part of the new Web UI under settings, and allows 3 options - No charging (same as old web ui), charging at a set rate (also like the old web ui) and Track Green Energy, which like the old Web UI, does not currently do anything but does update the settings.

Stage 2 which I'll hopefully test and commit later today is an updated policy which charges at nonScheduledAmpsMax where 1 or 2 are set, or falls back to Track Green Energy if nonScheduledAction is 3. 

It would be possible to backport this to the old Web UI, but my personal preference is to see it and the WebIPC protocol gone. That said, if someone was attached enough to the old UI to submit a PR for it, I'd merge it.
